renderdoc/driver/vulkan/vk_overlay.cpp

```diff
@@ -1347,7 +1347,7 @@ ResourceId VulkanReplay::RenderOverlay(ResourceId texid, FloatVector clearCol, D
           viewport.width = scissor.z;
           viewport.height = scissor.w;
 
-          vt->CmdSetViewport(Unwrap(cmd), 0, 1, &viewport);
+          vt->CmdSetViewport(Unwrap(cmd), 0, (uint32_t)state.views.size(), &viewport);
```

Comment:
Both of these changes seem extremely broken as you're passing potentially a number of viewports to the function but only passing one stack variable, meaning it will read off the end and read garbage. What was this change intended to do?